### DISCUSSION TITLE
lost user and group

### DISCUSSION THREAD

**Discussion by ducvu91** [minio/minio#14691] [Q&A] [answered]
Hello, i got issue lost all group and user when stop server minio. 
When I integration with S3 gateway. i create user and group, afther that, i stop and start again server minio. I lost all.
When i didn't integration with S3 gateway, it's normal
I using minio latest
Thanks

**Comment by klauspost** [comment, marked as answer]
https://docs.min.io/docs/minio-sts-quickstart-guide.html

See "Setup MinIO Gateway with Keycloak and Etcd" and https://github.com/minio/minio/blob/master/docs/sts/etcd.md
